### PR TITLE
adding dicom-viewer to staging.midrc envi

### PR DESCRIPTION
**Pull request by rnerella92**
Link to Jira ticket if there is one:

### Description of changes
Adding DICOM-viewer to staging environments

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# staging.midrc.org/manifest.json
## :warning: Services on branch
- dicom-viewer
- dicom-server

**Review by paulineribeyre — APPROVED** [review]
lgtm - waiting for approval from project side

**Pull request merged**
